dev-resources/bootstrap.cfg

```diff
@@ -0,0 +1,4 @@
+puppetlabs.services.version.version-check-service-web-service/hello-web-service
+puppetlabs.services.version.version-check-service-service/hello-service
```

Comment:
This is also left over from the TK template, so this file shouldn't actually exist. I'll delete it, sorry about that.